src/clients/python/api_v1/library/crequest.cc

```diff
@@ -247,13 +247,9 @@ ServerStatusContextGetServerStatus(
   ni::ServerStatus server_status;
   nic::Error err = ctx->ctx->GetServerStatus(&server_status);
   if (err.IsOk()) {
-    if (server_status.SerializeToString(&ctx->status_buf)) {
-      *status = &ctx->status_buf[0];
-      *status_len = ctx->status_buf.size();
-    } else {
-      err = nic::Error(
-          ni::RequestStatusCode::INTERNAL, "failed to parse server status");
-    }
+    ctx->status_buf = server_status.ShortDebugString();
```

Comment:
`ShortDebugString()` and `SerializeToString()` are not the same, the prior is stored in terms of text while the later is stored in terms of byte. Why do you have to make this change?

Reply:
This was breaking due to an issue in protobuf on Jetson. I spoke to @deadeyegoodwin about this workaround. Since it does not break backward compatibility it should be fine.